include/cocaine/rpc/queue.hpp

```diff
@@ -90,8 +90,10 @@ struct frozen_visitor:
 
 template<class Tag, class Upstream>
 class message_queue {
+public:
     typedef Upstream upstream_type;
```

Comment:
My fault. I was thinking about making non template attach with type defined in message_queue but changed my mind and did not change public back to private.